src/js/components/charts/SegmentedProgressBar.js

```diff
@@ -0,0 +1,51 @@
+import classNames from 'classnames';
+import React from 'react';
+
+import StackedProgressBar from './StackedProgressBar';
+
+class SegmentedProgressBar extends React.Component {
+  getSegment(segment, index) {
+    return (
+      <StackedProgressBar className={this.props.stackedProgressBarClassName}
+        progressState={segment.upgradeState} key={index}
+        secondaryLabel={segment.label} />
+    );
+  }
+
+  render() {
+    let props = this.props;
+    let classes = classNames(props.className);
```

Comment:
Oh oops, I noticed this afterwards, but I don't believe we need `classNames` here (or the `let classes` at all probably)